dropshot_endpoint/src/lib.rs

```diff
@@ -54,14 +54,19 @@ struct Metadata {
 
 const DROPSHOT: &str = "dropshot";
 
-const ERR_MSG_USAGE: &str = "Try using the following endpoint format:
-    async fn my_endpoint(
+fn usage(err_msg: &str, fn_name: &str) -> String {
```

Comment:
thanks for making this change. it looks like it boogered up the tests a tiny bit, but I think it's a bit nicer.